app/workers/functions.php

```diff
@@ -257,8 +258,18 @@ private function execute(
         $execution->setAttribute('status', 'processing');
         $execution = $dbForProject->updateDocument('executions', $executionId, $execution);
 
+        $vars = [];
+
+        $variables = $dbForProject->find('variables', [
```

Comment:
Same, variables should be available in the function document with the new subquery filter.